scripts/morph_router.py

```diff
@@ -0,0 +1,173 @@
+# coding=utf-8
+# Copyright 2025 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import asyncio
+from fastapi import FastAPI
+from pydantic import BaseModel, ConfigDict
+from typing import Optional, List
+from fastapi import FastAPI, Request
+import uvicorn
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+class BatchRequest(BaseModel):
+    """
+    BatchRequest is a data model representing a batch processing request.
+
+    Attributes:
+        scripts (list[str]): A list of script names or paths to be executed.
+        language (str): The programming language in which the scripts are written.
+        timeout (int): The maximum allowed execution time for each script in seconds.
+        request_timeout (int): The maximum allowed time for the entire batch request in seconds.
+    """
+    scripts: List[str]
+    language: str
```

Comment:
One assumption I made when I wrote the E2B router was that all scripts were in the same language, should we relax this assumption here?